src/main/java/graph/SequenceNode.java

```diff
@@ -37,4 +45,27 @@ public boolean isDummy() {
     public void setDummy(boolean dummy) {
         isDummy = dummy;
     }
+
+    public int getIndex() {
+        return this.index;
+    }
+
+    public void setIndex(int index) {
+        this.index = index;
+    }
+
+    /**
+     * Gets the column of the node.
+     * @return the column of the node.
+     */
+    public int getColumn() {
```

Comment:
Do you mind updating the unit tests that were on master so that they now cover these parts?